faker/providers/geo/de_AT/__init__.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from .. import Provider as GeoProvider
+
+
+class Provider(GeoProvider):
+
+    def latitude(self):
```

Comment:
didn't we decide for this to be `local_latitude`?

Reply:
Oops, yes, missed that one. Now fixed.